CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### New features
 
+## 1.0.0 - 2020-11-04
+
+There is no breaking change in this release, but from no on, as per semantic versioning,
+minor changes won't ever introduce breaking changes. 
```

Comment:
```suggestion
we will bump the major version when we change our publicly documented interface.
```

I'd be less adamant about breaking changes never occurring: it's a communication of intentions, rather than a guarantee of mistakes never happening :) (And also to emphasise that this only holds for publicly documented API's, e.g. we can change `getClientAuthnWithDependencies`.)

Reply:
Mistakingly introducing breaking changes in non-major releases ? That doesn't sound like us.